src/Control/Effect/Resumable.hs

```diff
@@ -88,7 +90,7 @@ runResumable :: ResumableC err m a -> m (Either (SomeError err) a)
 runResumable = runError . runResumableC
 
 newtype ResumableC err m a = ResumableC { runResumableC :: ErrorC (SomeError err) m a }
-  deriving (Alternative, Applicative, Functor, Monad, MonadFail, MonadIO, MonadPlus, MonadTrans)
+  deriving (Alternative, Applicative, Functor, Monad, MonadBase b, MonadFail, MonadIO, MonadPlus, MonadTrans, MonadTransControl)
```

Comment:
Because `Resumable` is sugar for `Error` over an existentially-quantified type, we can yield a `MonadTransControl` instance for it. We cannot do the same for `ResumableWith`, as it can cause nonlocal jumps.